compiler/damlc/tests/daml-test-files/SetFieldParens.EXPECTED.renamed-daml

```diff
@@ -1,4 +1,5 @@
-SetFieldParens.foo : (Text -> (Int, Bool)) -> (Int, Bool)
-SetFieldParens.foo mkTup
-= (setField
-@"_2" (not (getField @"_2" (mkTup "baz"))) (mkTup "bar"))
+  SetFieldParens.foo : (Text -> (Int, Bool)) -> (Int, Bool)
+  SetFieldParens.foo mkTup
+    = (setField
+         @"_2" (not (getField @"_2" (mkTup "baz"))) (mkTup "bar"))
+  [0m
```

Comment:
I know what happened here, I'll submit a PR to fix it

Reply:
https://github.com/digital-asset/daml/pull/16379, though I'll have to update it once this one is merged.